test/lib/creation.go

```diff
@@ -583,11 +584,15 @@ func (c *Client) CreatePodOrFail(pod *corev1.Pod, options ...func(*corev1.Pod, *
 
 	c.applyAdditionalEnv(&pod.Spec)
 
-	err := reconciler.RetryUpdateConflicts(func(attempts int) (err error) {
+	// the following retryable errors are expected when creating a blank Pod:
+	// - update conflicts
+	// - "No API token found for service account %q,
+	//    retry after the token is automatically created and added to the service account"
```

Comment:
Nit. I'd probably change the formatting of the comment a little bit. But that's just my opinion:
```
	// The following retryable errors are expected when creating a blank Pod:
	// - Update conflicts
	// - No API token found for service account, 
 	//   retry after the token is automatically created and added to the service account
```
Otherwise looks good.

Reply:
Well, it's quoted, because it refers to the specific error message literal as written in k8s, https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/serviceaccount/admission.go#L445